integrations/test_trec_covid_r3.py

```diff
@@ -0,0 +1,68 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import hashlib
+import os
+import re
+import shutil
+import unittest
+from random import randint
+from pyserini.util import download_url
+
+
+class TestSearchIntegration(unittest.TestCase):
+    def setUp(self):
+        self.round3_runs = {
+            'https://raw.githubusercontent.com/castorini/anserini/master/src/main/resources/topics-and-qrels/qrels.covid-round3-cumulative.txt':
+                'dfccc32efd58a8284ae411e5c6b27ce9',
+            'https://raw.githubusercontent.com/castorini/anserini/master/src/main/resources/topics-and-qrels/qrels.covid-round4-cumulative.txt':
+                '7a5c27e8e052c49ff72d557051825973'
+        }
+        self.tmp = f'../trec-covid-r3/check_data'
```

Comment:
put in `runs/`? Add a random number to directory to reduce change of collisions.